presto-docs/src/main/sphinx/release/release-0.247.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.247
+=============
+
+**Highlights**
```

Comment:
I can think of 2 highlights - the JDBC interceptor one and session scoped functions (@rongrong do you think it is worth a highlight?)